app/graphql/schema.rb

```diff
@@ -15,8 +15,8 @@ class Schema < GraphQL::Schema
   mutation Types::Mutation
   lazy_resolve(Promise, :sync)
   use GraphQL::Batch
-  use GraphQL::Execution::Interpreter
-  use GraphQL::Analysis::AST
+  # For batch-loading (see https://graphql-ruby.org/dataloader/overview.html)
+  # use GraphQL::Dataloader
```

Comment:
Do we plan to enable this in a soon future?

Reply:
Yes!